tests/test_op13_schema_traits_validation.py

```
@@ -3391,6 +3391,56 @@ def test_start(self):
+            {
+                "type": "object",
+                "x-gts-traits": {"indexed": None},
+            },
+            "register descendant trying to delete indexed",
+        ),
+        _validate_type_schema(
+            "gts.x.test13.mconstdel.event.v1~x.test13._.kid.v1~",
+            False,
+            "validate descendant - required prevents const-lock deletion",
+        ),
+    ]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)